ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -15,23 +15,30 @@
     </div>
   {{/if}}
 
-  <h1 class="title" data-test-title>
-    {{model.name}}
-    {{#if model.isConnectProxy}}
-      {{proxy-tag class="bumper-left"}}
-    {{/if}}
-    <span class="{{unless model.isConnectProxy "bumper-left"}} tag {{model.stateClass}}" data-test-state>{{model.state}}</span>
-    {{#if model.isRunning}}
-      {{two-step-button
-        data-test-restart
-        idleText="Restart"
-        cancelText="Cancel"
-        confirmText="Yes, Restart"
-        confirmationMessage="Are you sure? This will restart the task in-place."
-        awaitingConfirmation=restartTask.isRunning
-        disabled=restartTask.isRunning
-        onConfirm=(perform restartTask)}}
-    {{/if}}
+  <h1 class="title with-flex" data-test-title>
+    <div>
+      {{model.name}}
+      {{#if model.isConnectProxy}}
+        {{proxy-tag class="bumper-left"}}
+      {{/if}}
+      <span class="{{unless model.isConnectProxy "bumper-left"}} tag {{model.stateClass}}" data-test-state>{{model.state}}</span>
+    </div>
+    <div>
+      {{#if model.isRunning}}
+        <div class="two-step-button">
+          {{exec/open-button job=model.task.taskGroup.job taskGroup=model.task.taskGroup allocation=model.allocation task=model.task}}
```

Comment:
My rule of thumb is admittedly arbitrary, so feel free to disagree, but I think this component has enough props to break them out onto their own lines.

Reply:
agreed, that was OOC, I fixed it, thanks.